PowerUp/app/src/main/java/powerup/systers/com/utils/CSVReader.java

```diff
@@ -0,0 +1,112 @@
+package powerup.systers.com.utils;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import powerup.systers.com.data.entities.Accessory;
+import powerup.systers.com.data.entities.Answer;
+import powerup.systers.com.data.entities.Clothes;
+import powerup.systers.com.data.entities.Hair;
+import powerup.systers.com.data.entities.Question;
+import powerup.systers.com.data.entities.Scenario;
+
+public class CSVReader {
```

Comment:
Please check comments on this file here:
https://github.com/systers/powerup-android/pull/1137/files